src/app/portlets/dot-edit-page/dot-edit-page-routing.module.ts

```diff
@@ -2,6 +2,10 @@ import { PageViewResolver } from './dot-edit-page-resolver.service';
 import { NgModule } from '@angular/core';
 import { DotEditLayoutComponent } from './layout/dot-edit-layout/dot-edit-layout.component';
 import { RouterModule, Routes } from '@angular/router';
+import {
+    DotLegacyTemplateAdditionalActionsComponent
+// tslint:disable-next-line:max-line-length
+} from './layout/dot-template-additional-actions/dot-legacy-additional-actions-iframe/dot-legacy-template-additional-actions-iframe.component';
```

Comment:
The name of the folder of the component it's not consistent:

```dot-legacy-additional-actions-iframe``` **should be** ```dot-legacy-template-additional-actions-iframe``` right?